app/components/common/navbar/index.tsx

```diff
@@ -96,14 +98,17 @@ const Navbar = () => {
 
   return (
     <nav
-      className={`w-full md:w-52 ${classShortHands.flexDesktopCol} md:h-screen justify-between  ${classShortHands.padding} sm:px-4 border-t-2  md:border-r-2  border-stone-200 ${classShortHands.navbarBackgroundColor} fixed bottom-0 md:static`}
+      className={`w-full md:w-52 ${classShortHands.flexDesktopCol} md:h-screen justify-between  ${classShortHands.padding} sm:px-4 border-t-2  md:border-r-2  border-stone-200 ${classShortHands.navbarBackgroundColor} fixed bottom-0 md:static z-10`}
     >
+      <Slider isOpen={showSlider} toggle={setShowSlider} />
+
       <div className="basis-full">
         <ul
-          className={`${classShortHands.flexDesktopCol}   justify-evenly ${classShortHands.padding} py-2  mt-0 md:mt-7 md:text-sm md:font-medium ${classShortHands.navbarBackgroundColor}`}
+          className={`${classShortHands.flexDesktopCol} justify-evenly ${classShortHands.padding} py-2  mt-0 md:mt-7 md:text-sm md:font-medium ${classShortHands.navbarBackgroundColor}`}
         >
-          {navbarElements.navbarPages.map((navIntem: NavItemType) => (
+          {navbarElements.navbarPages.map((navIntem: NavItemType, index: number) => (
             <li
+              key={index}
```

Comment:
we can use the `id` available in `navItem` instead of index

Reply:
Done.